toplevel/topprinters.mli

```diff
@@ -13,5 +13,11 @@
 (*                                                                        *)
 (**************************************************************************)
 
-let () =
-  Topcommon.load_topdirs_signature ()
+(* Infrastructure to support user-defined printers in toplevels and debugger *)
+
+type printer_type = Types.type_expr -> Types.type_expr
```

Comment:
I'm not convinced that this name makes sense. I would remove the definition and inline its use sites.